treatment-service/integration-test/docker-compose/kafka.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3'
```

Comment:
Confluent Kafka Docker-compose setup copied from https://github.com/confluentinc/cp-all-in-one/blob/7.0.1-post/cp-all-in-one-kraft/docker-compose.yml.

Reply:
Would be nice to add this as inline comment in the code so it's easy to identify in the future.